scripts/rollup/modules.js

```diff
@@ -20,6 +20,7 @@ const importSideEffects = Object.freeze({
 const knownGlobals = Object.freeze({
   react: 'React',
   'react-dom': 'ReactDOM',
+  'interaction-tracking': 'InteractionTracking',
```

Comment:
We'll need to manually test that UMD bundles of ReactDOM in profiling mode work with UMD builds of IT.

Reply:
Yes. I created a fixture locally to test this, actually. I can add it to the repo.